src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/AppointmentsService.kt

```diff
@@ -101,9 +108,13 @@ class AppointmentsService(
       prisoner.checkForDuplicateAppointment(AppointmentType.VLB_PROBATION)
     }
 
+    val prison = prisonRepository.findByCode(prisoner.prisonCode)
+      ?.also { if (!user.isUserType(UserType.PRISON)) require(it.enabled) { "Prison with code ${it.code} is not enabled" } }
```

Comment:
Same here in terms of name and error message?  Just thinking of people who pick this up in future.